src/System.Windows.Forms/src/System/Windows/Forms/Printing/PrintPreviewDialog.cs

```diff
@@ -1248,7 +1248,14 @@ void InitForm()
             separatorToolStripSeparator1,
             closeToolStripButton});
             toolStrip1.Name = "toolStrip1";
-            toolStrip1.RenderMode = System.Windows.Forms.ToolStripRenderMode.System;
+
+            // Note: ToolStripSystemRenderer provides not enough contrasted color
```

Comment:
[nit] a bit more English-sounding:
```suggestion
            // in High Contrast mode the color scheme provided by ToolStripSystemRenderer 
            // is not sufficiently contrast; so disable it.
```

Reply:
Thanks!